src/openrct2-ui/windows/Ride.cpp

```diff
@@ -619,17 +619,27 @@ struct VehicleTypeLabel
     const char* label_string;
 };
 
+// Used for sorting the entrance type dropdown.
+struct EntranceTypeLabel
+{
+    ObjectEntryIndex EntranceTypeId;
+    StringId LabelId;
+    const char* label_string;
```

Comment:
Could you rename this to `LabelString`? We use TitleCase for struct members.

Reply:
That's the name and data type used in the other sorting-related structs, `RideTypeLabel` and `VehicleTypeLabel`.  
Shall I update them as well then?

Reply:
Renamed to TitleCase; also did that for the other 2 structs.  
Also wrapped it in `u8string_view`. Again, same for the other structs.

Reply:
Thank you, that is a nice cleanup.